<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HallWidget</class>
 <widget class="QWidget" name="HallWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>640</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>640</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>640</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="BackToLogin">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>10</y>
     <width>100</width>
     <height>71</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>50</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/resource/image/windows/close.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="PushButton_Ranking">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>100</width>
     <height>70</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>70</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>70</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/resource/image/windows/rank.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="PushButton_StartGame">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>560</y>
     <width>141</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/resource/image/windows/startButton.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>70</y>
     <width>550</width>
     <height>350</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>550</width>
     <height>350</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>550</width>
     <height>350</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QTableView {
    color: white;                                       /*表格内文字颜色*/
    gridline-color: black;                              /*表格内框颜色*/
    background-color: rgb(108, 108, 108);               /*表格内背景色*/
    alternate-background-color: rgb(64, 64, 64);
    selection-color: white;                             /*选中区域的文字颜色*/
    selection-background-color: rgb(77, 77, 77);        /*选中区域的背景色*/
    border: 2px groove gray;
    border-radius: 0px;
    padding: 2px 4px;
}
QHeaderView {
    color: white;
    font: bold 10pt;
    background-color: rgb(108, 108, 108);/*设置表头空白区域背景色*/
    border: 0px solid rgb(144, 144, 144);
    border:0px solid rgb(191,191,191);
    border-left-color: rgba(255, 255, 255, 0);
    border-top-color: rgba(255, 255, 255, 0);
    border-radius:0px;
    min-height:29px;
}
QHeaderView#hHeader::section {
    background-color:darkcyan;
    color: red;padding-left: 4px;
    border: 1px solid #6c6c6c;
    height:40;
}


QHeaderView#vHeader::section {
    background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,stop:0 #616161, stop: 0.5 yellow,stop: 0.6 green, stop:1 cyan);
    color: white;padding-left: 4px;border: 1px solid #6c6c6c;
    width:30;
}
QHeaderView::section {
    color: white;
    background-color: rgb(64, 64, 64);
    border: 5px solid #f6f7fa;
    border-radius:0px;
    border-color: rgb(64, 64, 64);
} </string>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QPushButton" name="PushButton_knight">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>250</y>
     <width>231</width>
     <height>311</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/resource/image/character/Knight/stop.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="PushButton_Magician">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>260</y>
     <width>231</width>
     <height>311</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
border-image: url(:/resource/image/character/Magician0.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QListView" name="listView">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>1011</width>
     <height>641</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/resource/image/windows/bac2.jpg);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PushButton_ResumeGame">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>470</y>
     <width>141</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(resource/image/windows/resume.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>listView</zorder>
  <zorder>BackToLogin</zorder>
  <zorder>PushButton_Ranking</zorder>
  <zorder>PushButton_StartGame</zorder>
  <zorder>PushButton_knight</zorder>
  <zorder>PushButton_Magician</zorder>
  <zorder>tableView</zorder>
  <zorder>PushButton_ResumeGame</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
